client/src/ui/BitbakeTerminal.ts

```diff
@@ -27,7 +27,7 @@ export async function runBitbakeTerminalCustomCommand (bitbakeDriver: BitbakeDri
 }
 
 const bitbakeTerminals: BitbakeTerminal[] = []
-async function runBitbakeTerminalScript (command: string, bitbakeDriver: BitbakeDriver, terminalName: string, bitbakeScript: string, isBackground: boolean): Promise<IPty> {
+export async function runBitbakeTerminalScript (command: string, bitbakeDriver: BitbakeDriver, terminalName: string, bitbakeScript: string, isBackground: boolean): Promise<IPty> {
```

Comment:
It looks like you are using runBitbakeTerminalCustomCommand so no need for a new export?

Reply:
removed.